File: web-portal/src/app/model/auth-response.ts
export class AuthResponse {
    private name:string = "";
    private projectName:string = "";
    private isValid:boolean = false;

    public get Name():string{
        return this.name;
    }

    public get ProjectName():string{
        return this.projectName;
    }

    public get IsValid():boolean{
        return this.isValid;
    }

    public set Name(name:string){
        this.name=name;
    }

    public set ProjectName(pname:string){
        this.projectName=pname;
    }

    public set IsValid(value:boolean){
        this.isValid = value;
    }

}

File: Severity/Severity/src/main/java/com/cognizant/Severity/model/AuthResponse.java
package com.cognizant.Severity.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AuthResponse {
    private int empId;
    private String name;
    private String projectName;
    private boolean isValid;
}
